### PR TITLE
Improved Adminhelp sound

### PR DESCRIPTION
**Pull request by VloBoo**
## About the PR
Replaced the adminhelp sound with a more attractive sound of a falling pipe

## Why / Balance
During the chaos around April 1, people may not notice that an admin is writing to them. Therefore, I believe that the sound needs to be improved so that everyone can hear it clearly.

## Technical details
I hope I didn't make any stupid mistakes in the PR.

## Media
https://github.com/space-wizards/space-station-14/assets/60321021/3e7d293e-5fdc-42e8-b5d8-1a9fa0bf26b7
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
**Changelog**
:cl:
- tweak: Improved Adminhelp sound!

### CODE REVIEW

File: Resources/Audio/Effects/attributions.yml [hunk 1]
```diff
@@ -216,3 +216,8 @@
   copyright: Bhijn and Myr (github username deathride58)
   license: CC0-1.0
   source: https://github.com/space-wizards/space-station-14/pull/23212
+
+- files: ["metal-pipe-falling.ogg"]
+  copyright: "Taken from thenotcheeseman via freesound.org and mixed from stereo to mono."
+  license: CC0-1.0
+  source: "https://freesound.org/people/thenotcheeseman/sounds/679206/"
```
Comment (metalgearsloth) [inline Resources/Audio/Effects/attributions.yml:223]: I'm reasonably sure this is not licensed properly and is a stolen sound effect from the original pack from the 90s.

**Comment by CatMagic1** [comment]
new player experience improved by 110% on april fools day

**Comment by FairlySadPanda** [comment]
Sloth please just this once 

please

**Comment by CatMagic1** [comment]
i too will beg if necessary

File: Resources/Audio/Effects/attributions.yml (hunk 1 above)
Comment (VloBoo) [reply Resources/Audio/Effects/attributions.yml:228]: Unfortunately, I could not find another original. (＞︿＜)
Is this source definitely not suitable as the original?

**Comment by MaloTV** [comment]
Oh god please i require this

**Comment by Slava0135** [comment]
as i figured out in #26078 metal pipe sound can't be used

**Pull request closed without merging**